test/parse/locator.test.js

```diff
@@ -28,76 +15,98 @@ describe('DOMLocator', () => {
 			'  -->',
 			'',
 			'  <state id="start" name="start">',
-			'    <transition event"init" name="init" target="main_state" />',
+			'    <transition event="init" name="init" target="main_state" />',
```

Comment:
It looks like you fixed a piece of the XML, which is good. But I think it would be a nice to test a piece of XML with `=` missing as well.

Reply:
:eyes: :+1: 
I think when I "fixed the XML" I discovered that the same XML was also present in one of the error test files. But I removed that part later since it was not used for testing anywhere. So this is related to your other comment about converting it into a test.

I will look into it again.

Reply:
The one with missing `=` is now part of `xml-error.test.js`